tests/analyze/test_plot_correlation.py

```
@@ -0,0 +1,132 @@
+from readii.analyze.correlation import getFeatureCorrelations
+from readii.io.writers.plot_writer import PlotWriter
```

Ruff (0.8.2)

`readii.io.writers.plot_writer.PlotWriter` imported but unused

Remove unused import: `readii.io.writers.plot_writer.PlotWriter`

(F401)